script.js

```diff
@@ -1,88 +1,73 @@
-let books = [
+let books = [];
+
+if (books === []) {
+  books = [
     {
-   titulo: '100 años de soledad',
-   author: 'Gabriel Garcia Marquez'
+      titulo: '100 años de soledad',
+      author: 'Gabriel Garcia Marquez',
     },
     {
-   titulo: 'El tirano',
-   author: 'Massimo Manfredy'
-    }
-]
-
-const bookList = document.querySelector('.books-display');
-let displ = ''
-books.forEach((el, index) => {    
-    displ += `
-    <div class="oneBook" id="book${index}">
-    <h4> ${el.titulo} </h4>
-    <h4> ${el.author} </h4> 
-    <button class="btn-display" id="${index}">remove</button> 
-    </div>
-    ` 
-
-});
-bookList.innerHTML = displ
-let removIds = [...document.querySelectorAll('.btn-display')].map(item => item.id)
-
-const newB = document.getElementById('bk')
+      titulo: 'El tirano',
+      author: 'Massimo Manfredi',
+    },
+  ];
+}
 
-newB.addEventListener('submit', function(event){
+function storage(ok) {
+  window.localStorage.setItem('localBooks', JSON.stringify(ok));
+}
 
-    event.preventDefault();
-    const reciTi = document.getElementById("recibe-ti").value
-    const reciAu = document.getElementById("recibe-au").value
+function getBooks() {
+  books = JSON.parse(window.localStorage.getItem('localBooks'));
+}
 
-    if(reciTi !== '' && reciAu !== ''){
-    let newBook = {
-        titulo: reciTi,
-        author: reciAu
-    }
-    books.unshift(newBook)
+getBooks();
 
-    let displ = ''
-    books.forEach((el, index) => {    
+function display() {
+  const bookList = document.querySelector('.books-display');
+  let displ = '';
+  books.forEach((el, index) => {
     displ += `
     <div class="oneBook" id="book${index}">
     <h4> ${el.titulo} </h4>
     <h4> ${el.author} </h4> 
     <button class="btn-display" id="${index}">remove</button> 
     </div>
-    `
-
-    });
-    bookList.innerHTML = displ
-
-    console.log(books)
+    `;
+  });
+  bookList.innerHTML = displ;
 }
-})
-
-/* document.querySelectorAll('.btn-display').forEach( (rem) => {
-    rem.addEventListener('click', (event) => {
-        console.log(rem)
-    })
 
-}) */
+// ---------ADD-------------
+
+const newB = document.getElementById('bk');
+newB.addEventListener('submit', (event) => {
+  event.preventDefault();
+  const reciTi = document.getElementById('recibe-ti').value;
+  const reciAu = document.getElementById('recibe-au').value;
+
+  if (reciTi !== '' && reciAu !== '') {
+    const newBook = {
+      titulo: reciTi,
+      author: reciAu,
+    };
+    books.unshift(newBook);
+    storage(books);
+    display();
+  }
```

Comment:
- [ ] Like the previous reviewer suggested, your add and remove functionality don't work yet as shown in the image below 👇. Please fix it to move to the next milestone 👍 
![image](https://user-images.githubusercontent.com/46839548/182447270-8944adc5-c238-4611-a470-ea815bb8826f.png)